server/src/routes/root.js

```diff
@@ -1,8 +0,0 @@
-const example = require('./example.route');
-
-const express = require('express');
-const router = express.Router();
-
-router.use('/example', example);
-
-module.exports = router;
```

Comment:
Don't delete this file as it's the root file in which all the routes will come under and then this root file will be returned to the index.js.

Your auth.route.js will also return to here not index.js

Reply:
I forgot to push